selenium/test_app.py

```diff
@@ -0,0 +1,178 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.options import Options
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.chrome.service import Service as ChromeService
+from selenium.webdriver.common.keys import Keys
+import time
+
+chrome_options = Options()
+options = [
+  "--headless",
+  "--disable-gpu",
+  "--window-size=1920,1200",
+  "--ignore-certificate-errors",
+  "--disable-extensions",
+  "--no-sandbox",
+  "--disable-dev-shm-usage"
+]
+
+for option in options:
+  chrome_options.add_argument(option)
+
+se = ChromeService(ChromeDriverManager().install())
+driver = webdriver.Chrome(service=se,options=chrome_options)
+
+# make sure the main page loads and contains the necessary elements
+def test_main_page():
+  url = "http://localhost:3000" # Change to port 3000 if running locally
```

Comment:
We will be running it using a docker container.

```suggestion
  url = "http://localhost" # Change to port 3000 if running locally
```

Reply:
did it